### ISSUE TITLE
anvil-outline

### ISSUE THREAD

**Issue by DWallinTresensa** [Templarian/MaterialDesign#7311]
### Before submitting a new icon contribution, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://pictogrammers.com/library/mdi) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](https://pictogrammers.com/docs/library/mdi/guides/brand-icons/) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I have followed the [Material guidelines](https://material.io/design/iconography/system-icons.html) for system icons.

### Tell us about your contribution.

I wanted an alternative to the existing anvil icon with a lighter outlined style

### Share a preview of your contribution.

![image](https://github.com/Templarian/MaterialDesign/assets/28813210/9bc27043-100c-4f40-b961-bb1b50acd388)

### Upload the icon source.

[anvil-outline.svg.zip](https://github.com/Templarian/MaterialDesign/files/13514831/anvil-outline.svg.zip)

**Comment by MrGrigri** [comment]
I believe the horn should also have a 2 pixels wide stroke

**Comment by DWallinTresensa** [comment]
The material guidelines suggests "If an icon requires complex details, subtle adjustments can be made to improve its legibility. " and gives two examples of icons using 1.5px, the paperclip (entirely 1.5px) and the ramen bowl (mixed 2px and 1.5px). I started with entirely 2px, but it makes the interior of that shape overly closed and throws off the balance of the icon so I moved it to 1.5px. You can see what I mean here: 
![image](https://github.com/Templarian/MaterialDesign/assets/28813210/13e99ef8-8780-40fe-90f7-cf7238f59b0c)

**Comment by DWallinTresensa** [comment]
I'm also attaching a comparison with the solid version. I tried to keep it visually consistent but I think it's useful to highlight differences.
![image](https://github.com/Templarian/MaterialDesign/assets/28813210/dfa39929-5e3e-42f9-9f8b-da0a499674a9)

**Comment by MrGrigri** [comment]
Yes...but it also doesn't allow to mix the two; both 1px and 2px spacing in the same icon. What if the horn was filled in. And we also try to keep the two consistent with shapes as they are sometimes toggles on UX. So when a user clicks the icon there will be a slight shift. Sometimes this is unavoidable. 

Don't get me wrong, I thank you for your efforts and I like how you are thinking this through. And it may result in us changing the default `anvil` icon to match the outline of this one.

**Comment by DWallinTresensa** [comment]
I was following the guidelines here but realize they might not exactly match the guidelines used within this community:
https://m3.material.io/styles/icons/designing-icons
<img width="529" alt="Screen Shot 2023-12-04 at 11 32 14 AM" src="https://github.com/Templarian/MaterialDesign/assets/28813210/36063a45-65ca-46ca-8c05-69abf875909a">

Is there another guide you recommend?

**Comment by MrGrigri** [comment]
Or just keeping the horn solid could also work.

![image](https://github.com/Templarian/MaterialDesign/assets/4528551/ebd5c4ba-880b-4012-87ac-1b35a6bb96c2)

**Comment by DWallinTresensa** [comment]
I think the solid horn approach works, but if reworking the existing anvil is in the cards I would suggest the following:

![image](https://github.com/Templarian/MaterialDesign/assets/28813210/f2eab33a-ca6c-4eb7-99f4-1b59711e9f1e)
![image](https://github.com/Templarian/MaterialDesign/assets/28813210/b2980321-3000-4540-914d-eb1d4d6127e7)

It reworks the curves to utilize 45° angles and circular arcs that are more consistent with the Material look.

**Comment by DWallinTresensa** [comment]
Attaching files for these alternate icons
[anvil_rework.svg.zip](https://github.com/Templarian/MaterialDesign/files/13550553/anvil_rework.svg.zip)
[anvil-outline_rework.svg.zip](https://github.com/Templarian/MaterialDesign/files/13550554/anvil-outline_rework.svg.zip)

**Comment by DWallinTresensa** [comment]
Another option if we are more willing to break with the existing general silhouette. This better sticks to recommended icon sizes/dimensions, and it's better centered visually, but is a bigger departure from the existing icon:
![image](https://github.com/Templarian/MaterialDesign/assets/28813210/2e9c77d6-4561-48d9-b49e-e57a16f3ecb7)
![image](https://github.com/Templarian/MaterialDesign/assets/28813210/5a579db5-422e-486c-8209-1b5bb2b3df29)

and files:
[anvil_reworkB.svg.zip](https://github.com/Templarian/MaterialDesign/files/13561929/anvil_reworkB.svg.zip)
[anvil-outline_reworkB.svg.zip](https://github.com/Templarian/MaterialDesign/files/13561930/anvil-outline_reworkB.svg.zip)
